Danger-Swift/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+import WeTransferLinterTests
+
+var tests = [XCTestCaseEntry]()
```

Comment:
Does it make sense to make this available on Linux as our CI setup is specialised for iOS? In that case we could specify the targets in the `Package.swift` file e.g. like this: 
```
platforms: [.macOS(.v10_12),
                                  .iOS(.v10),
                                  .tvOS(.v10),
                                  .watchOS(.v3)],
```

Reply:
You might be right! I'll update that

Reply:
I did remove the `LinuxMain.swift` but did not add the platforms as it does not make sense. Our package has nothing related to those OS versions as it's a CI tool.

Reply:
Is our CI tool not only related to iOS and Apple platforms? I am trying to understand how the swift package manager works here. I might be wrong but if we don't specify these platforms doesn't it mean that our package will still be available to linux for example? Maybe there is a way to exclude the linux platform instead of specifying all the other platforms. Let me know what you think.

Reply:
Our CI tool is related to iOS and Apple platforms in the sense that it's _testing_ those platforms. The CI tool defined as a package can be run from both Linux and Mac machines, checking out the GIT repository there.

In that case, it would make more sense to define the platforms as `.linux` and `.macOS` only. 
However, we can't define Linux, so I rather keep this out of our Package definition. This is also the common way to handle Danger-Swift packages afaik.

If we would create an iOS library like WeScan, you would define the iOS platforms that it supports. This, however, is a CI library/tool and it should not contain those platforms.

Let me know if that makes more sense to you.

Reply:
Thanks for explaining!  Strange though that we can't exclude to run on linux, because that would not work since we depend on Xcode. I agree that for this reason leaving it out of there is the best solution!